styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/MetricsStats.java

```diff
@@ -60,43 +59,43 @@
 
   private static final MetricId STORAGE_RATE = BASE
       .tagged("what", "storage-operation-rate")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "operation");
 
   private static final MetricId DOCKER_DURATION = BASE
       .tagged("what", "docker-operation-duration")
-      .tagged("unit", UNIT_MILLISECOND);
+      .tagged("unit", "operation");
 
   private static final MetricId DOCKER_RATE = BASE
       .tagged("what", "docker-operation-rate")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "operation");
 
   private static final MetricId TRANSITIONING_DURATION = BASE
       .tagged("what", "time-transitioning-between-submitted-running")
       .tagged("unit", UNIT_SECOND);
 
   private static final MetricId PULL_IMAGE_ERROR_RATE = BASE
       .tagged("what", "pull-image-error-rate")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "error");
 
   private static final MetricId NATURAL_TRIGGER_RATE = BASE
       .tagged("what", "natural-trigger-rate")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "triggering");
 
   private static final MetricId TERMINATION_LOG_MISSING = BASE
       .tagged("what", "termination-log-missing")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "count");
 
   private static final MetricId TERMINATION_LOG_INVALID = BASE
       .tagged("what", "termination-log-invalid")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "count");
 
   private static final MetricId EXIT_CODE_MISMATCH = BASE
       .tagged("what", "exit-code-mismatch")
-      .tagged("unit", UNIT_FREQUENCY);
+      .tagged("unit", "count");
```

Comment:
As this is used for a meter, `count` would be emitted as `count/s` which doesn't seem to make much sense. How about e.g. `mismatch` instead? We could also omit the unit and fall back to the default of `n` which is emitted as `n/s`.

Similar for the rest of the units.

https://github.com/spotify/semantic-metrics#meter